OptimizelySDK/Odp/OdpManager.cs

```diff
@@ -248,20 +216,33 @@ public Builder WithCacheImplementation(ICache<List<string>> cache)
             /// <returns>OdpManager instance</returns>
             public OdpManager Build(bool asEnabled = true)
             {
+                _logger = _logger ?? new DefaultLogger();
+                _errorHandler = _errorHandler ?? new NoOpErrorHandler();
+
+                if (_odpConfig == null)
```

Comment:
What about setting `if (odpConfig == null) _odpConfig = new ODPConfig()` default. 
Initially that's the best they can do when initializing ODPManager.